### ISSUE TITLE
What does "Have ballot, will vote" mean?

### ISSUE THREAD

**Issue by abrauninger** [VotingVoices/chorus#100]
I'm not sure what this step is telling me to do:

![image](https://user-images.githubusercontent.com/43097655/47140901-4cb88300-d274-11e8-8a7f-30a52c78ddef.png)

Should we simplify the wording and say something more direct about sending your ballot?

**Comment by seattlehedgehog** [comment]
Actually, now that i see this in action, I wonder if we need a step for the 'yes' in the plan at all. There is not a call to action and the step below tells you to send it or drop it off.

**Comment by abrauninger** [comment]
Removed the step for the "yes" answer in commit adea916.
